test/integration/test/stream.js

```diff
@@ -0,0 +1,82 @@
+const {AudioConfig} = require('../../../dist/src/models/streaming/AudioConfig');
+const {RevAiStreamingClient} = require('../../../dist/src/streaming-client');
+const configHelper = require('../src/config-helper');
+const fs = require('fs');
+const assert = require('assert');
+const colors = require('colors');
+
+(async (done) => {
+    const audioConfig = new AudioConfig("audio/x-raw", 'interleaved', 16000, 'S16LE', 1);
+    const client = new RevAiStreamingClient(configHelper.getApiKey(), audioConfig);
+    client.baseUrl = `wss://${configHelper.getBaseUrl()}/speechtotext/v1alpha/stream`;
+        
+    client.on('close', (code, reason) => {
+        console.log(`Connection closed, ${code}: ${reason}`);
+        assertCloseCodeAndReason(code, reason);
+        printPassStatement();
+        return;
+    });
+    client.on('httpResponse', code => {
+        console.log(`Streaming client received http response with code: ${code}`);
+    });
+    client.on('connectFailed', error => {
+        console.log(`Connection failed with error: ${error}`);
+    });
+    client.on('connect', connectionMessage => {
+        console.log(`Connected with job id: ${connectionMessage.id}`);
+    });
+
+    var stream = client.start();
+
+    stream.on('data', data => {
+        if (data.type === 'partial') {
+            assertPartialHypothesis(data);
```

Comment:
Will this test still pass if we don't get any partials or finals? I feel like it will

Reply:
Good catch. It will fail but later down the road during the close assertions but that would not be an accurate failure. I can add an else conditional here that explicitly fails the test in the case neither are present.